Remove fingerprint for better perf

roberttod opened the pull request:
- The fingerprint is great but it is slow, especially because it checks cookie.
- Use random int twice and add an incrementor to the timestamp in order to spice it up a little bit

roberttod commented:
@alanclarke @jimmed (maybe also this function could be moved into slapdash)

alanclarke commented:
readme is a lie

alanclarke commented on really-unique-id.js:
```diff
@@ -33,20 +36,7 @@ function hammerTime () {
-  var maxVal = Math.pow(2, 32) - 1
-  var fingerPrint = navigator.userAgent + document.cookie
-  return format(hash(fingerPrint), maxVal)
+  return format((new Date()).getTime() + incrementor++, doomsDay.getTime())
```
not sure if doomsday adds any uniqueness? Like its a fixed number of ms from now, but new Date().getTime() is also a fixed number of ms from now? So they're like the same number, but with a slightly different offset

Commit:
Remove fingerprint for better perf
- The fingerprint is great but it is slow, especially because it checks cookie.
- Use random int twice and add an incrementor to the timestamp in order to spice it up a little bit

Commit:
Update README.md

alanclarke commented:
:+1:
